.github/workflows/main.yml

```diff
@@ -140,33 +140,172 @@ jobs:
         os: [ubuntu-18.04]
     steps:
       - uses: actions/checkout@v2
-      - name: Set up bazel
-        uses: jwlawson/actions-setup-bazel@v1
-        with:
-          bazel-version: '3.4.1'
       - name: Clone Oppia Bazel
         run: git clone https://github.com/oppia/bazel.git $HOME/oppia-bazel
       - name: Set up JDK 9
         uses: actions/setup-java@v1
         with:
           java-version: 9
-      - name: Extract Android Tools
+      - name: Extract Android tools
         run: |
           mkdir -p $GITHUB_WORKSPACE/tmp/android_tools
           cd $HOME/oppia-bazel
           unzip bazel-tools.zip
           tar -xf $HOME/oppia-bazel/android_tools.tar.gz -C $GITHUB_WORKSPACE/tmp/android_tools
-      - name: Unzip Bazel Binary and Build Oppia
+      # See https://git-secret.io/installation for details on installing git-secret. Note that the
+      # apt-get method isn't used since it's much slower to update & upgrade apt before installation
+      # versus just directly cloning & installing the project. Further, the specific version
+      # shouldn't matter since git-secret relies on a future-proof storage mechanism for secrets.
+      # This also uses a different directory to install git-secret to avoid requiring root access
+      # when running the git secret command.
+      - name: Install git-secret (non-fork only)
+        if: github.repository == 'oppia/oppia-android'
+        shell: bash
+        run: |
+          cd $HOME
+          mkdir -p $HOME/gitsecret
+          git clone https://github.com/sobolevn/git-secret.git git-secret
+          cd git-secret && make build
+          PREFIX="$HOME/gitsecret" make install
+          echo "$HOME/gitsecret" >> $GITHUB_PATH
+          echo "$HOME/gitsecret/bin" >> $GITHUB_PATH
+      - name: Decrypt secrets (non-fork only)
+        if: github.repository == 'oppia/oppia-android'
+        env:
+          GIT_SECRET_GPG_PRIVATE_KEY: ${{ secrets.GIT_SECRET_GPG_PRIVATE_KEY }}
+        run: |
+          cd $HOME
+          # NOTE TO DEVELOPERS: Make sure to never print this key directly to stdout!
```

Comment:
It looks like this line needs a tab back to be a comment, right now I guess it's not working as a comment. Or it's just the IDE!!

<img width="907" alt="Screenshot 2020-12-04 at 14 35 10" src="https://user-images.githubusercontent.com/25057618/101144187-f0747100-363d-11eb-956c-2469468ae4be.png">
<img width="909" alt="Screenshot 2020-12-04 at 14 35 01" src="https://user-images.githubusercontent.com/25057618/101144198-f4a08e80-363d-11eb-810e-a426a79af05a.png">

Reply:
It actually is being "executed" as a comment as far as I can tell (the ``run`` block just executes everything in a shell, and ``#`` conventionally is ignored by shells). I actually lean toward keeping the code as-is because it's the echo line that's specifically sensitive and needs care taken when it's changed.

Reply:
Ok